publisher-provider/publisher-provider-app/src/main/java/nl/idgis/publisher/provider/database/messages/PerformCount.java

```diff
@@ -2,22 +2,41 @@
 
 import nl.idgis.publisher.database.messages.Query;
 
-public class PerformCount extends Query {		
+public class PerformCount extends Query {
 
-	private static final long serialVersionUID = 8499023090069062665L;
+	private static final long serialVersionUID = 1464334851511334051L;
 	
 	private final String tableName;
+
+	private final String scheme;
 	
 	public PerformCount(String tableName) {
-		this.tableName = tableName;
+
+		String[] schemaTableParts = tableName.split("\\.");

```

Comment:
Dit is inmiddels de derde keer dat ik dit Java fragment zie terugkomen. Dit vraagt om een utility class.

Reply:
Samen naar gekeken. Te gevolgen daarvan zijn niet makkelijk te overzien.